dev-python/pipenv/pipenv-2021.5.29-r1.ebuild

```diff
@@ -20,11 +20,17 @@ KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
 IUSE="test"
 RESTRICT="!test? ( test )"
 
-PATCHES="${FILESDIR}/${PN}-${PV//./-}-${PR}-remove-attr-vendor-import.patch"
+PATCHES=( 
+	"${FILESDIR}/${PN}-${PV//./-}-${PR}-remove-attr-vendor-import.patch"
+	"${FILESDIR}/${PN}-${PV//./-}-${PR}-remove-colorama-vendor-import.patch"
+	)
 
 RDEPEND="
 	${PYTHON_DEPS}
+	dev-python/attr[${PYTHON_USEDEP}]
```

Comment:
As CI say, we have no `dev-python/attr` in the tree.

But we do have a `dev-python/attrs`, I think it's the right thing? Please double check.

Reply:
Darn, the package name is different from the module imported. Sorry about it.